sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -848,6 +852,45 @@ def execute_query_raw(self, query):
             cursor.execute(query)
             return cursor.fetchall()
 
+    # https://learn.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-db-index-usage-stats-transact-sql?view=sql-server-ver15
+    def collect_index_usage_metrics(self):
+        if not is_affirmative(self.instance.get('include_index_usage_metrics', True)):
+            return False
+
+        now = time.time()
+        if now - self.index_usage_last_check_ts > INDEX_USAGE_STATS_INTERVAL:
+            self.log.warning('Collecting index usage statistics')
```

Comment:
The `warning` is just for debugging I take it? Just change to .debug before merge.